build.properties.sample

```diff
@@ -10,5 +10,17 @@ build.bootclasspath = ${jdk7.home}/jre/lib/rt.jar\
     :${jdk7.home}/jre/lib/jsse.jar\
     :${jdk7.home}/jre/lib/jce.jar
 
-# Classpath of JDBC drivers for "ant run".
-jdbc.classpath =
+# Classpath of JDBC drivers for "ant run" and "ant test".
+# jdbc.classpath = path/to/JDBC/driver
+
+# By default, the unit tests use an in-memory H2 database.
+# However, you can configure a different database to use for the
+# tests here.
+
+# Supported databases: H2, MySQL, Oracle, SQLServer
+# test.jdbc.database = MySQL
```

Comment:
MySQL is the least interesting of these. If we're going to have these, we should include sample driver names and URLs for all three. Not a fan of the "jdbc" in the names; other than "database" the names are adaptor-config.properties names, so maybe "test.database" and "test.db.*" if there's concern about clashes.